.github/workflows/run-acceptance-tests.yml

```diff
@@ -1,380 +1,142 @@
-# WARNING: This file was build based on https://github.com/pulumi/ci-mgmt
-# but it is currently not in sync. Any update must be done manually
+# WARNING: This file is autogenerated - changes will be overwritten when regenerated by https://github.com/pulumi/ci-mgmt
 
 name: run-acceptance-tests
+
 on:
   pull_request:
-    branches:
-    - main
     paths-ignore:
     - CHANGELOG.md
   repository_dispatch:
     types:
     - run-acceptance-tests-command
+
 env:
-  REPOSITORY: github://api.github.com/equinix
-  PROVIDER: equinix
   PR_COMMIT_SHA: ${{ github.event.client_payload.pull_request.head.sha }}
-  DOTNETVERSION: 7.0.x
-  JAVAVERSION: "11"
-  NODEVERSION: 20.x
-  PYTHONVERSION: "3.9"
-  GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
-  SLACK_WEBHOOK_URL: ${{ secrets.SLACK_WEBHOOK_URL }} # TODO
+  PULUMI_API: https://api.pulumi-staging.io
+  PULUMI_GO_DEP_ROOT: ${{ github.workspace }}/..
+  PULUMI_LOCAL_NUGET: ${{ github.workspace }}/nuget
+  PULUMI_PROVIDER_AUTOMATION_TOKEN: ${{ secrets.PULUMI_PROVIDER_AUTOMATION_TOKEN }}
   TF_APPEND_USER_AGENT: pulumi
-  # Allow missing or extra mappings due to TF provider
-  # upgrades.  This allows automation to do its job so
-  # that human contributions can be narrowly focused on
-  # adding or removing resource mappings
-  PULUMI_SKIP_MISSING_MAPPING_ERROR: true
-  PULUMI_SKIP_EXTRA_MAPPING_ERROR: true
```

Comment:
We manually added these `PULUMI_SKIP_` environment variables to a couple of workflows to smooth out the upgrade process.  Having these in all workflows would be bad--we don't want to release broken providers--but it doesn't look like `pulumi/ci-mgmt` gives the kind of granular control we would need to only have these environment variables in some workflows but not in others.  It's probably better to lose these if it means we can follow a standard development process.